.travis.yml

```diff
@@ -9,6 +9,8 @@ branches:
 
 install:
     - pip install -U tox codecov
+    - pip install proxy.py
+    - proxy.py --port 8123 &
```

Comment:
It works on python3, just did not think about adding it there. Now I start a twisted proxy from tests, so just ``tox`` works.
Now I like it much better, thanks!